qcodes/instrument_drivers/Newport/AG_UC8.py

```diff
@@ -171,10 +244,20 @@ class Newport_AG_UC8(VisaInstrument):
             for example "ASRL3" for COM3.
     """
 
+    # By default, expect response to command within 1 second.
     default_timeout = 1.0
+
+    # Position scan commands (and absolute move commands) can take
+    # up to 2 minutes to complete.
     position_scan_timeout = 120.0
-    command_delay = 0.002   # wait 2 ms between commands
-    reset_delay = 0.05      # wait 50 ms after reset command
+
```

Comment:
in order to not introduce confusion, could you at least mention here that "slow" command is a position scan command (or better, rename the `_slow_command` method to `_position_scan_command`. Just so that we have a single term for this case, not two.